app/services/analytics_events.rb

```diff
@@ -1640,6 +1640,12 @@ def idv_doc_auth_submitted_pii_validation(
     )
   end
 
+  # User visits IdV verify step waiting on a resolution proofing job result
+  # @identity.idp.previous_event_name IdV: doc auth verify visited
+  def idv_doc_auth_verify_polling_wait_visited(**extra)
+    track_event(:idv_doc_auth_verify_polling_wait_visited, **extra)
```

Comment:
if there's no arguments ever used I think we can drop `**extra` I think

Reply:
I hadn't noticed that pattern on the other events like that in here. Willfix!

Reply:
This is relevant to some work I'm doing as well and I am 👍 on excluding `**extra` if possible